candy-kata/README.md

```diff
@@ -0,0 +1,34 @@
+<!--
+  ~ Copyright (c) 2020 The Bank of New York Mellon.
+  ~ All rights reserved. This program and the accompanying materials
+  ~ are made available under the terms of the Eclipse Public License v1.0
+  ~ and Eclipse Distribution License v. 1.0 which accompany this distribution.
+  ~ The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+  ~ and the Eclipse Distribution License is available at
+  ~ http://www.eclipse.org/org/documents/edl-v10.php.
+  -->
+# **What is the Candy Kata?**
+The Candy Kata is an intermediate Code Kata with a set exercises that a developer can complete to familiarize themselves with many of the APIs available in [Eclipse Collections](https://github.com/eclipse/eclipse-collections).
+
+The domain of the Candy Kata includes the `Candy` and `SchoolGroup` classes.
+
+<a href="http://eclipse.github.io/eclipse-collectioans-kata/company-kata/#/1/15"><img src="https://github.com/eclipse/eclipse-collections-kata/blob/master/docs/company-kata/company-domain.png" height="50%" width="50%"></a>
```

Comment:
I believe you meant to update the href and the img src to be the candy-domain image since it's part of this PR, but you still have the company-domain linked here.

Reply:
Updated